pages/1.11/installing/ent/custom/system-requirements/index.md

```diff
@@ -30,8 +30,6 @@ Here are the master node hardware requirements.
 
```

Comment:
Add the following text, before the table "For supported Operating Systems and Environments are listed on https://docs.mesosphere.com/version-policy/"